FlowCrypt/src/main/java/com/flowcrypt/email/api/retrofit/ApiService.kt

```diff
@@ -143,7 +149,7 @@ interface ApiService {
    * @param body POJO model for requests
    */
   @POST(BuildConfig.API_URL + "account/login")
-  suspend fun postLogin(@Body body: LoginModel, @Header("Authorization") tokenId: String):
+  suspend fun postLogin(@Body body: LoginModel, @Header("Authorization") idToken: String):
```

Comment:
I'd rename here and in other methods of this class `idToken` into `authorization`, because it actually contains not only token itself but the whole content of the header.

Reply:
Agree :+1: